src/tools/sdk/index.md

```diff
@@ -20,6 +20,8 @@ A package manager can help you easily install and update the Dart SDK.
 Don’t want to use a package manager? Other options are
 [building the SDK from source][] and
 [downloading the SDK as a zip file](/tools/sdk/archive).
+If you use either of these options, remember to
```

Comment:
I feel like even if you use a package manager, you might need to alter your path... but let's see how this flies.